### DISCUSSION TITLE
Логика размещения постов

### DISCUSSION THREAD

**Discussion by TorinAsakura** [atls/kino-reporter#3] [General]
Надо расписать какие посты и по какому приоритету в какой блок вы постите.

К примеру, сейчас на главной первый блок с большой картинкой и стопкой, справа, поменьше рендерят посты из разных категорий.

Хочется понять есть ли какая-то стратегия или приоритеты.

**Comment by AlexeyGovorukhin** [comment]
Так, ну сейчас всё относительно просто:
под меню  "слайдер" - большая картинка и справа 4 маленькие
большая картинка закрепляется вручную в меню редактора чекбоксом "показывать первой"
в правой колонке слайдера - две новости и два закрепленных материала на красном фоне (закрепляется вручную в меню редактора чекбоксом "закрепить"). 
Ниже в ленте в правой колонке статьи (всё, кроме новостей), в левой колонке - только новости
